src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -103,7 +107,19 @@ public IReadOnlyCollection<Mutant> GetLatestMutantBatch()
         /// <returns>Mutated node</returns>
         public SyntaxNode Mutate(SyntaxNode currentNode)
         {
-            return Mutate(currentNode, new MutationContext(this));
+            var mutationContext = new MutationContext(this);
+            var mutation = Mutate(currentNode, mutationContext);
+            if (mutationContext.HasStatementLevelMutant && _options?.DevMode == true)
+            {
+                Log.Error($"Several mutants were not injected in the project : {mutationContext.BlockLevelControlledMutations.Count+mutationContext.StatementLevelControlledMutations.Count}");
```

Comment:
This uses serilog directly instead of through Microsoft.Extensions.Logging.ILogger, skipping our configured logger.